src/app/users/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,54 @@
+<h3 i18n>Member Profile</h3>
+<form [formGroup]="editForm" (ngSubmit)="onSubmit()">
+  <mat-grid-list cols="3">
```

Comment:
Please use `<div>` tags instead.  The `mat-grid-list` isn't doing anything without `mat-grid-tile` as child components, and the layout will not be good without doing a lot of work to tweak the grid attributes.